de.metas.adempiere.adempiere/base/src/main/java/org/adempiere/ad/service/impl/LookupDAO.java

```diff
@@ -808,6 +810,17 @@ protected NamePair fetch(final ResultSet rs) throws SQLException
 			final String name = getDisplayName(rs, isActive);
 			final String description = rs.getString(MLookupFactory.COLUMNINDEX_Description);
 
+			String validationMessage;
+			try
+			{
+				validationMessage = rs.getString(MLookupFactory.COLUMNINDEX_ValidationMsg);
+			}
+			catch (Exception e)
+			{
+				validationMessage = null;
+				logger.info("Only used in DocAction dropdown");
```

Comment:
whether the value is set only in DocAction dropdown should not be this code's concern

Reply:
exactly.
more, avoid `log.info` for repetitive stuff. use log.debug/trace